unit_test/CMakeLists.txt

```diff
@@ -382,7 +412,9 @@ IF (KOKKOS_ENABLE_THREADS)
     batched_dla_threads
     SOURCES
       Test_Main.cpp
-      threads/Test_Threads_Batched_Dense.cpp
+      threads/Test_Threads_Batched_Dense_Serial_Other.cpp
```

Comment:
I would prefer that these be broken into separate a single cpp file for each kernel, like you've done for Sparse. This sets future maintainers up for following this pattern to avoid regressing our build times.